tests/test_core.py

```diff
@@ -9,28 +9,31 @@
 from faker import Faker
 from singer_sdk.testing import get_tap_test_class, suites
 from singer_sdk.testing.runners import TapTestRunner
-from sqlalchemy import Column, DateTime, Integer, MetaData, Numeric, String, Table, text
+from sqlalchemy import BigInteger, Column, DateTime, MetaData, Numeric, String, Table, text
 from sqlalchemy.dialects.postgresql import BIGINT, DATE, JSON, JSONB, TIME, TIMESTAMP
 from test_replication_key import TABLE_NAME, TapTestReplicationKey
 from test_selected_columns_only import (
     TABLE_NAME_SELECTED_COLUMNS_ONLY,
     TapTestSelectedColumnsOnly,
 )
 
-from tap_postgres.tap import TapPostgres
+from tap_cratedb.tap import TapCrateDB
+
+# DB_SCHEMA_NAME = "public"
+DB_SCHEMA_NAME = "doc"
```

Comment:
Used to switch default schema from `public` (PostgreSQL) to `doc` (CrateDB). Makes sense to contribute to upstream [tap-postgres](https://github.com/MeltanoLabs/tap-postgres).